web/app.py

```diff
@@ -0,0 +1,101 @@
+from flask import Flask, render_template, request
+# pip install flask-socketio
+# pip install gevent-websocket
+from flask_socketio import SocketIO
+import can
+import cantools
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = 'secret!'
+socketio = SocketIO(app,cors_allowed_origins="*")
+
+# Set up bus
+bustype = 'socketcan'
+channel = 'vcan0'
+
+bus = can.interface.Bus(bustype=bustype, channel=channel)
+
+# Get data from CAN2
+db = cantools.database.load_file("can_data/CAN2.dbc")
+user10_msg = db.get_message_by_name("USER_10")
```

Comment:
literal constants like USER_10 are fine because the whole point of the webpage is to use these exact messages. They will never, ever change.